frontend/ui/widget/doublespinwidget.lua

```diff
@@ -54,7 +54,9 @@ local DoubleSpinWidget = InputContainer:new{
 function DoubleSpinWidget:init()
     self.screen_width = Screen:getWidth()
     self.screen_height = Screen:getHeight()
-    self.width = self.width or math.floor(math.min(self.screen_width, self.screen_height) * 0.6)
+    if not self.width or self.width <= 1 then
+        self.width = math.floor(math.min(self.screen_width, self.screen_height) * (self.width or 0.8))
+    end
```

Comment:
I don't think I really mind this `self.width` 's doubling meaning/purpose (a real width when > 1, a factor when <=1), but it feels a bit quick and dirty (I think I had done something like that for testing an idea, and I had pushed it in a PR, but @Frenzie had stopped me :)
So, let's be sure of the right way to do it, before you spread the idea to other widgets.

Reply:
Examples of both to make the final decision:
https://github.com/koreader/koreader/blob/67ba76eca0e725b3773a622ac961e107b8a88557/frontend/ui/data/creoptions.lua#L167
https://github.com/koreader/koreader/blob/0ea8bee4e1fa4324999d08ae0865f96b01407202/plugins/gestures.koplugin/main.lua#L455
The latter is `Gesture interval`, where there is a lot of help text and the widget is high.
It doesn't fit in the landcape with just the factor of 0.75, so it needs to be widened.

Reply:
The comment proposed
https://github.com/koreader/koreader/blob/67ba76eca0e725b3773a622ac961e107b8a88557/frontend/ui/widget/spinwidget.lua#L27

Reply:
Well, ok, as we can't find an ubiquitous name for when it's a factor between 0 and 1, let's just keep it "width" all around, and the comments should be enough.
So:
- in creoptions.lua: `widget_width = 0.6,`
- `width = more_options_param.widget_width,`
- and update the comment ("which is smaller" reads not well, you probably meant "whichever is smaller" ?), ex:
`-- width, or when between 0 and 1, factor to the smallest of screen width and height`

Reply:
I definitely prefer the _factor one! That one's just unclear, while plain width is deceptive. But having to look it up is better than being deceived. ;-)